qt5script: Check for QtGui before building the examples
Signed-off-by: Vicente Olivert Riera <Vincent.Riera@imgtec.com>

From e4453c92ee41943a3a1d67a725804a44e747f64e Mon Sep 17 00:00:00 2001
From: Vicente Olivert Riera <Vincent.Riera@imgtec.com>
Date: Wed, 16 Apr 2014 12:19:17 +0100
Subject: [PATCH] qtscript: Check for QtGui before building the examples

qsdbg and marshal examples need QtGui module. If QtGui is not installed
they fail to compile.

Change-Id: I1af6a8b39a0848c8208fe94a26ee57409dfd9b41
Reviewed-by: Vicente Olivert Riera <Vincent.Riera@imgtec.com>
Reviewed-by: Allan Sandfeld Jensen <allan.jensen@digia.com>
---
 examples/script/script.pro | 9 ++++-----
 1 file changed, 4 insertions(+), 5 deletions(-)

diff --git a/examples/script/script.pro b/examples/script/script.pro
index 3e1eae6..cc97233 100644
--- a/examples/script/script.pro
+++ b/examples/script/script.pro
@@ -1,7 +1,7 @@
 TEMPLATE = subdirs
-SUBDIRS = \
-    customclass \
-    qsdbg
+SUBDIRS = customclass
+
+qtHaveModule(gui): SUBDIRS += qsdbg
 
 qtHaveModule(widgets) {
     SUBDIRS += \
@@ -22,8 +22,7 @@ qtHaveModule(widgets) {
 }
 
 !wince {
-    SUBDIRS += \
-        marshal
+    qtHaveModule(gui): SUBDIRS += marshal
 }
 
 maemo5: CONFIG += qt_example
-- 
1.9.1

